### ISSUE TITLE
 The loss turns nan when continue training on pre_trained model ?

### ISSUE THREAD

**Issue by zimurui** [pierluigiferrari/ssd_keras#217]
Thank for your code. And  I trained the ssd300 on VOC dataset for about 40 epochs, but was interrupted due to a sudden restart of the computer. Now I want to load the pretrained weights and continue training. But the loss trun nan in the 4th batch(shown below), and callbacks terminate the train. The code shows bellow:
 
` # model.load_weights('VGG_ILSVRC_16_layers_fc_reduced.h5', by_name=True)   comment
model.load_weights(ssd300_pascal_07+12_epoch-41_loss-5.0485_val_loss-5.0070.h5,by_name=True)`

<img width="552" alt="default" src="https://user-images.githubusercontent.com/32056332/51391476-1a693b00-1b6d-11e9-8780-7e74fff56b0d.PNG">
